components/brave_wallet_ui/common/async/lib.ts

```diff
@@ -976,11 +949,21 @@ export async function sendEthTransaction (payload: SendEthTransactionParams) {
     }
     // @ts-expect-error google closure is ok with undefined for other fields but mojom runtime is not
     const txDataUnion: BraveWallet.TxDataUnion = { ethTxData1559: txData1559 }
-    addResult = await apiProxy.txService.addUnapprovedTransaction(txDataUnion, payload.fromAccount.address, null, null)
```

Comment:
This is intended. `addUnapprovedTransaction` will soon accept `fromAccount`.

Reply:
fixed